cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,66 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  const balance = depositAmount - withdrawAmount + 5096;
```

Comment:
The calculation of `balance` is incorrect because `depositAmount` and `withdrawAmount` are strings. You should convert them to numbers before performing arithmetic operations. Consider using `Number(depositAmount)` and `Number(withdrawAmount)`.